sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityFactorModelReader.java

```diff
@@ -21,23 +15,16 @@
 public class SensitivityFactorModelReader implements SensitivityFactorReader {
 
     private final List<SensitivityFactor> factors;
-    private final Network network;
 
-    public SensitivityFactorModelReader(List<SensitivityFactor> factors, Network network) {
+    public SensitivityFactorModelReader(List<SensitivityFactor> factors) {
         this.factors = Objects.requireNonNull(factors);
-        this.network = Objects.requireNonNull(network);
     }
 
     @Override
     public void read(Handler handler) {
         Objects.requireNonNull(handler);
         for (SensitivityFactor factor : factors) {
             String functionId = factor.getFunctionId();
-            if (factor.getFunctionType() == SensitivityFunctionType.BUS_VOLTAGE) {
-                Bus bus = new IdBasedBusRef(factor.getFunctionId()).resolve(network, TopologyLevel.BUS_BRANCH)
```

Comment:
In current code we can also through IdBasedBusRef specify bus using an injection or branch ID: so we get rid of this ?